lib/workers/repository/init/merge.spec.ts

```diff
@@ -33,6 +35,9 @@ jest.mock('../../../config/migrate-validate');
 
 describe(getName(), () => {
   describe('detectRepoFileConfig()', () => {
+    beforeEach(async () => {
+      await initialize({});
+    });
     it('returns config if not found', async () => {
```

Comment:
```suggestion

    it('returns config if not found', async () => {
```

Can we add newlines before each test, it makes easier readable